docs/mkdocs/documentation/install.md

```diff
@@ -2,8 +2,18 @@
 
 ## Using `kubectl`
 
+First install [cert-manager](https://github.com/cert-manager/cert-manager) which is a dependency.
 ```shell
-kubectl apply -k https://github.com/kubernetes-sigs/kernel-module-management/config/default
+kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.11.0/cert-manager.yaml
+kubectl -n cert-manager wait --for=condition=Available deployment \
+	cert-manager \
+	cert-manager-cainjector \
+	cert-manager-webhook
+```
+
+Install KMM.
+```shell
+kubectl apply -k https://github.com/kubernetes-sigs/kernel-module-management/config/kmm
```

Comment:
Why don't we revert to `config/default`?

Reply:
Because now we have 2 operators installing, `cert-manager` and KMM so I though it make more sense to call it `kmm`.